src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Rename/FileMoveNotificationListener.cs

```diff
@@ -137,7 +137,9 @@ async Task<List<(Renamer.RenameDocumentActionSet, string)>> GetNamespaceUpdateAc
 
                     // This is a file item to another directory, it should only detect this a Update Namespace action.
                     // TODO Upgrade this api to get rid of the exclamation sign
+#pragma warning disable CS0618 // Type or member is obsolete
                     Renamer.RenameDocumentActionSet documentAction = await Renamer.RenameDocumentAsync(oldDocument, null!, documentFolders);
+#pragma warning restore CS0618 // Type or member is obsolete
```

Comment:
Roslyn has deprecated a bunch of these rename APIs.

@ocallesp do we have a work item tracking the update of this? If not, could you file one please?

Reply:
I filed https://github.com/dotnet/project-system/issues/8591